mobile/lib/modules/home/ui/asset_grid/immich_asset_grid.dart

```diff
@@ -28,6 +28,7 @@ class ImmichAssetGrid extends HookConsumerWidget {
   final bool showMultiSelectIndicator;
   final void Function(ItemPosition start, ItemPosition end)?
       visibleItemsListener;
+  final Widget? topWidget;
```

Comment:
Really like the idea of the topWidget. We can use that afterwards to improve the album asset grid view and get rid of the suboptimal nested scrolling :-)

Reply:
Yeah I was creating some NestedScrollView, and it get hairy quickly so I opt-in to find an easier solution :P